backend/otel/otel.go

```diff
@@ -224,6 +236,7 @@ func (o *Handler) HandleTrace(w http.ResponseWriter, r *http.Request) {
 								RequestID:       &requestID,
 								TraceID:         pointy.String(traceID),
 								SpanID:          pointy.String(spanID),
+								LogCursor:       pointy.String(clickhouse.EncodeCursor(ts, uuid)),
```

Comment:
This seems fine for now, but it does feel a bit like we're leaking private logic.
Maybe in the future we could have an accessor like "cursorFor(logRow)"

Mostly just trying to prevent the logic from spreading out. We can chat about this in our next 1-1z

Reply:
Actually, this would be pretty elegant:

```go
func (logRow LogRow) Cursor() string {
	return encodeCursor(logRow.Timestamp, logRow.UUID)
}
```

Reply:
makes sense! added a `LogRow.Cursor()` method for this